config/locales/es/activerecord.yml

```diff
@@ -315,6 +318,11 @@ es:
         code: "Código"
         title: "Título"
         description: "Descripción"
+      sdg/local_target:
+        code: "Código"
```

Comment:
A translation for the `target` (or is it `target_id`?) attribute is missing, and the form label says "Target" in Spanish.

Reply:
Done

Reply:
It still says "Target" :thinking:.

Reply:
My bad; wrong branch! :sweat_smile: